packer/provision/docker.sh

```diff
@@ -20,7 +36,7 @@ DOCKER_NETWORK_OPTIONS='--bip=10.250.0.254/24'
 EOL
     # configure docker daemon to listen on port 5555 enabling remote
     # managment
-    sed -i -e "s#='--selinux-enabled'#='--selinux-enabled -H unix:///var/run/docker.sock -H tcp://0.0.0.0:5555'#g" /etc/sysconfig/docker
+    sed -i -e "s#='--selinux-enabled'#='--selinux-enabled -H unix:///var/run/docker.sock -H tcp://0.0.0.0:5555'#g" /etc/sysconfig/docker-ce
```

Comment:
Are you sure that the sysconfig file changes it's name to docker-ce?

Reply:
I'm not sure...but `/etc/sysconfig/docker` doesn't exist

Reply:
Now docker recommends to put daemon config in `/etc/docker/daemon.yaml`.  The default location according to the docs.